codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyFrameCodec.java

```diff
@@ -403,6 +531,25 @@ public void readHeaderBlockEnd() {
         }
     }
 
+    public void readUnknownFrame(int frameType, byte flags, ByteBuf payload) {
+        read = true;
+        SpdyUnknownFrame spdyUnknownFrame = new DefaultSpdyUnknownFrame(frameType, flags, payload);
+        ctx.fireChannelRead(spdyUnknownFrame);
```

Comment:
how about adding a protected method: `createSpdyUnknownFrame`, then I can override it.

Reply:
Sure I can do this if you need it.